### ISSUE TITLE
Broken compatibility with Node.js 16.0

### ISSUE THREAD

**Issue by pubmikeb** [LinusU/fs-temp#5]
`fs‑temp` passes `null` as the path parameter: https://github.com/LinusU/fs-temp/blob/c892a4786eb7e6669076649ad9c6e2cb3631a037/lib/write-stream.js#L6 which breaks the compatibility with Node.js 16.0 and leads to the crash when using `fs‑temp`:

```js
node:internal/process/promises:245
          triggerUncaughtException(err, true /* fromPromise */);
          ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received null
    at TempWriteStream.WriteStream (node:internal/fs/streams:360:5)
    at new TempWriteStream (C:\Users\User\IdeaProjects\TMS\node_modules\fs-temp\lib\write-stream.js:6:15)
    at Object.createWriteStream (C:\Users\User\IdeaProjects\TMS\node_modules\fs-temp\lib\temp.js:121:10)
    at Busboy.<anonymous> (C:\Users\User\IdeaProjects\TMS\node_modules\multer\lib\read-body.js:70:27)
    at Busboy.emit (node:events:369:20)
    at Busboy.emit (C:\Users\User\IdeaProjects\TMS\node_modules\busboy\lib\main.js:37:33)
    at PartStream.<anonymous> (C:\Users\User\IdeaProjects\TMS\node_modules\busboy\lib\types\multipart.js:214:13)
    at PartStream.emit (node:events:369:20)
    at HeaderParser.<anonymous> (C:\Users\User\IdeaProjects\TMS\node_modules\dicer\lib\Dicer.js:50:16)
    at HeaderParser.emit (node:events:369:20) {
  code: 'ERR_INVALID_ARG_TYPE'
}

Process finished with exit code 1
```

At [Node.js thread](https://github.com/nodejs/node/issues/38058) I've got a response that the change https://github.com/nodejs/node/pull/35187 has broken some compatibility and `fs-temp` has to be adjusted to be able to work with Node.js 16.0+

**Update:**
Node.js 16.0 has been officially released and neither `fs‑temp`, nor any depending on it module don't support Node.js 16.0.

**Comment by pubmikeb** [comment]
@LinusU, after a brief codebase review, I assume that it is enough to pass `"null"` to `WriteStream.call(…)` to fix a broken compatibility with the Node.js 16.0. I've tried to propose a Pull Request, but I don't have permissions.

Could you, please, review this issue to ensure the support of Node.js 16.0 by `fs-temp`?
Thanks!

**Comment by pubmikeb** [comment]
@LinusU, I've prepared a fix how can I propose it for the project?
It looks like I don't have a permission to propose a PR.

Thanks.

**Comment by ExE-Boss** [comment]
You should just be able to [open a PR from your own fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork), which you need to create: <https://github.com/LinusU/fs-temp/fork>

**Comment by LinusU** [comment]
@pubmikeb what do you think about an empty string (`''`) instead of a string with the word "null"?

**Comment by pubmikeb** [comment]
> @pubmikeb what do you think about an empty string (`''`) instead of a string with the word "null"?

Right, an empty string (`''`) solves the issue, I've proposed a pull request.

**Comment by pubmikeb** [comment]
@LinusU, the pull request with the fix is proposed, please review it and in case everything is OK, the PR can be merged.
Hopefully, https://github.com/expressjs/express will update the dependencies shortly after that.

**Comment by LinusU** [comment]
Really sorry for the huge delay in this!!

I fixed this in f02bfe3 which is part of the [🚢 2.0.0 / 2021-07-24](https://github.com/LinusU/fs-temp/releases/tag/v2.0.0) release!
